dto/decision_dto.go

```diff
@@ -40,6 +40,14 @@ type APIDecisionRule struct {
 	RuleId        string    `json:"rule_id"`
 }
 
+type APIDecisionRuleWithDetail struct {
+	APIDecisionRule
+	RuleDetail struct {
+		FormulaAstExpression *NodeDto `json:"formula_ast_expression"`
+		ScoreModifier        int      `json:"score_modifier"`
```

Comment:
I mean, ok I get why it's there but I don't quite get how it works in this case.
You compose structs, so which of the two values does it take to fill in the json score_modifier field ? The second one just because it comes second ?